docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1612,6 +1612,119 @@ Note/limitations:
 
 - Applying a uniform non-response rate limits the impact of race/ethnicity, age, and sex to affect the sampled population. This might make some aspects of PRL easier as it is less likely the same simulants will be missing from each sample.
 
+Women, Infants, and Children (WIC)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+WIC is a government benefits program designed to support mothers and young 
+children. The main qualifications are income and presence of young children 
+in the home. 
+
+
+**When to Sample** 
+
+- Sample compiled on the time step containing Jan 1st of each year (the time step might end on Jan 2nd, Jan 15th, Jan 27th, etc.)
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Household number (random indicator which is the same for simulants in the same household)
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - Age 
+  * - DOB 
+  * - Home Address 
+  * - Home Zip Code 
+
+Here is an example: 
+
+.. image:: survey_example.png
+
+**Who to Sample** 
+
+Selection for the observer will be in 2 steps: 
+
+#. Eligibilty for WIC 
+#. Covered by WIC 
+
+Please note that multiple people in the same household can be selected and selection 
+should be independent (e.g., having one child in the household on WIC does not mean a 
+second child is or is not on WIC). Inclusion is at the simulant level. Do not include 
+others in the household on the observer. 
+
+**Eligibility for WIC**
+
+Eligibility for WIC is based on income and age/children in the house. 
+To qualify you must be both: 
+
+- A child under the age of 5 (0-4 years old)
+- OR a tracked mother OR guardian of a child under the age of 1 
+
+And also: 
+
+- Have a total household income less than the limits below (based on number of people in the household)
+
+.. list-table:: Income Maxs  
+  :widths: 10 10
+  :header-rows: 1
+
+  * - Number of people in household 
+    - Maximum Income 
+  * - 1 
+    - $25,142
```

Comment:
These are all evenly spaced. So, it would be simpler to say $16,410 + $8,732 * number of people in the household.

Reply:
Fair enough!